app/controllers/api/v4/identifiers_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V4::IdentifiersController < UnauthenticatedApiController
```

Comment:
i don't love this name, but the other names i considered didn't feel better:
* `AppIdentifiersController`
* `AppVersionBuildPathIdentifiersController`
* `AppVersionBuildPathIdentifierVisitorConfigsController`

🤮 

The main thing i don't love about this name is that it doesn't indicate that this controller is nestled within "app version build path" stuff. In other other controllers where we do this, we call the controller `app*` but `AppIdentifiers` feels like a confusing name, not a clarifying one. Given that, i decided to stick with just Identifiers.

Reply:
I think you should go with the App prefix, otherwise you've got two different version numbers of the same controller name that have different semantics, are for different client types, and are both the latest version of their API endpoint for their use case.